### ISSUE TITLE
Lets colors be complaint with WCAG 2.1 

### ISSUE THREAD

**Issue by yangwao** [kodadot/nft-gallery#1132]
The goal is to keep colours as least possible intrusive for the eye.

Play https://contrast-ratio.com/#%23ff149366-on-black
https://www.w3.org/TR/WCAG/#contrast-minimum

![image](https://user-images.githubusercontent.com/5887929/141615168-388d99b3-279e-4b0a-b762-f7dc5d7eedbf.png)

h/t @roiLeo noting this
